src/xdocs/config_filters.xml

```diff
@@ -2232,6 +2232,296 @@ public class UserService {
+          a separate suppressions file, which must be kept up-to-date as the source
+          file is edited.
+        </p>
+      </subsection>
+      <subsection name="Properties" id="SuppressWithNearbyTextFilter_Properties">
+        <div class="wrapper">
+          <table>
+            <tr>
+              <th>name</th>
+              <th>description</th>
+              <th>type</th>
+              <th>default value</th>
+              <th>since</th>
+            </tr>
+            <tr>
+              <td>nearbyTextPattern</td>
+              <td>Specify nearby text pattern to trigger filter to begin suppression.</td>
+              <td><a href="property_types.html#Pattern">Pattern</a></td>
+              <td><code>"SUPPRESS CHECKSTYLE (\w+)"</code></td>
+              <td>10.3.4</td>
+            </tr>
+            <tr>
+              <td>checkPattern</td>
+              <td>Specify check name pattern to suppress.</td>
+              <td><a href="property_types.html#Pattern">Pattern</a></td>
+              <td><code>".*"</code></td>
+              <td>10.3.4</td>
+            </tr>
+            <tr>
+              <td>messagePattern</td>
+              <td>Specify check violation message pattern to suppress.</td>
+              <td><a href="property_types.html#Pattern">Pattern</a></td>
+              <td><code>null</code></td>
+              <td>10.3.4</td>
+            </tr>
+            <tr>
+              <td>idPattern</td>
+              <td>Specify check ID pattern to suppress.</td>
+              <td><a href="property_types.html#Pattern">Pattern</a></td>
+              <td><code>null</code></td>
+              <td>10.3.4</td>
+            </tr>
+            <tr>
+              <td>influenceFormat</td>
+              <td>Specify negative/zero/positive value that defines the number of lines
+                  preceding/at/following the suppression comment.</td>
+              <td><a href="property_types.html#String">String</a></td>
+              <td><code>"0"</code></td>
+              <td>10.3.4</td>
+            </tr>
+          </table>
+        </div>
+      </subsection>
+      <subsection name="Examples" id="SuppressWithNearbyTextFilter_Examples">
+        <p>
+          To configure the filter to suppress audit events on the same line:
+        </p>
+        <source>
+&lt;module name=&quot;Checker&quot;&gt;
+  &lt;module name=&quot;SuppressWithNearbyTextFilter&quot;/&gt;
+  &lt;module name=&quot;TreeWalker&quot;&gt;
+    &lt;module name=&quot;ConstantNameCheck&quot;/&gt;
+  &lt;/module&gt;
+&lt;/module&gt;
+        </source>
+        <p>
+          Example:
+        </p>
+        <source>
+static final int lowerCaseConstant; // SUPPRESS CHECKSTYLE the reason for this is..
+static final int lowerCaseConstant1; // violation
+static final int lowerCaseConstant2; // violation
+static final int lowerCaseConstant3; // violation
+static final int lowerCaseConstant4; // violation
```

Comment:
can we minimize all code examples for user ?
to let user read quickly and focus on what matter quickly

Reply:
I personally wouldn't because nearly all of the example code snippets are same so it is easy to follow how behavior changes when different properties are tweaked. Moreover, the lines are same apart from variable name `lowerCaseConstant1` ->  `lowerCaseConstant2` so I do not find it that overwhelming/hard to look at.

But it might just be me because I wrote them down and you as a reviewer see something I can not.

That being said, let me know if you have a particular suggestion of how they can be shortened because I cannot think of any right now.

Reply:
@romani Done. Tried to shorten all examples as much as possible. Tell me what you think.